src/components/Jumbotron.js

```diff
@@ -15,6 +15,11 @@ export default function Jumbotron(props) {
           </h2>
           <p className="base--p jumbotron--description description">
             {props.description}
+            <br />
```

Comment:
Why don't we use `{props.description}` for this? 

Maybe it's better than hardcoding a string. If that's not the case we should add a variable and only show the GDPR message if the variable is true (false by default).
Here you are introducing a breaking change.

Reply:
I think it's better to keep update the {props.description} because end user can add the message that way. And we don't need to update React components that way.